lib/google_crawler/google/scrapper.ex

```diff
@@ -1,12 +1,74 @@
 defmodule GoogleCrawler.Google.Scrapper do
```

Comment:
I wondered from the beginning why you added two `p` to "scrapper". Only today, I checked you were right about it as I making some improvements to my implementation as well 🙈 

Well according to Google, scrapper is probably not what you are looking for:

![image](https://user-images.githubusercontent.com/696529/79313112-0ce66080-7f2a-11ea-9066-77b69db1af84.png)

While scraping/scraper is:

![image](https://user-images.githubusercontent.com/696529/79313187-27203e80-7f2a-11ea-91fd-6d6cb4a88eec.png)

Reply:
Wait, which one is correct 😂?? So It's `scraper`, right? Now I'm confused 😲

Reply:
Yes, scraper (the one below)

Reply:
I'll fix the name 😂 👍